STOREFRONT_LAB.md

```diff
@@ -0,0 +1,185 @@
+# Commerce Partner Days - EDS Storefront Session
+
+## Table of Contents
+- [Overview](#overview)
+- [What You'll Learn](#what-youll-learn)
+- [Prerequisites](#prerequisites)
+- [Important Links](#important-links)
+- [Payment Method Integration](#payment-method-integration)
+
+## Overview
+This lab will guide you through creating and integrating a custom payment method into your Adobe Commerce storefront, including setting up the payment processing logic and webhook subscriptions. You'll learn how to create a new payment method called "PARTNER-PAY" and implement the necessary backend and frontend components.
+
+## What You'll Learn
+- How to create and configure a custom payment method in Adobe Commerce
+- How to set up webhook subscriptions for payment validation
+- How to integrate payment method logic into the storefront
+- How to handle payment sessions and validation
+- Best practices for payment method implementation
+
+## Prerequisites
+1. **Adobe Commerce Access**
+   - Active Adobe Commerce sandbox environment
+   - Admin access to your sandbox
+   - Valid tenant ID
+
+2. **Development Environment**
+   - Terminal with `curl` and `jq` installed
+   - Adobe I/O CLI (`aio`) installed and configured
+   - Access to your Stage workspace
+   - Code editor (VS Code recommended)
+
+3. **Required Accounts**
+   - Adobe Commerce admin credentials
+   - Adobe I/O Console access
+   - Stage workspace access
+
+## Important Links
+After scaffolding your storefront, you'll have access to these important URLs:
+- **Storefront Preview**: `https://main--{repo}--{owner}.aem.page/`
+- **Content Editor**: `https://da.live/#/{owner}/{repo}`
+- **Configuration Manager**: `https://da.live/#/{owner}/{repo}/configs-stage`
+- **Admin URL**: `https://na1-sandbox.admin.commerce.adobe.com/{tenant_id}`
+- **REST API Endpoint**: `https://na1-sandbox.api.commerce.adobe.com/{tenant_id}`
+
+## Payment Method Integration
+
+### Step 1: Set Up Environment Variables
+1. Open your terminal
+2. Set your REST API endpoint (replace `{tenant_id}` with your actual tenant ID):
+```bash
+export REST_API=https://na1-sandbox.api.commerce.adobe.com/{tenant_id}
+```
+
+### Step 2: Generate and Set Access Token
+1. Go to your Stage workspace
```

Comment:
Maybe you can put the link to the developer console and the org that has to be selected as it is done in https://github.com/adobe-commerce/partner-day-accs/blob/0c74f3a1b3c8617f2d750933d1c032fbfce4df53/README.md?plain=1#L323